From 9dd7da4718fea6bc537f50db3e5b1a0a40941162 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Wed, 23 May 2018 16:04:19 +0800
Subject: [PATCH 3872/5242] MLK-18368-8: imx-hdmi: change the platform device
 id for hdmi audio tx

commit  a85446893be69fdcb1c648522dcfbc416b47a9c7 from
https://source.codeaurora.org/external/imx/linux-imx.git

platform device id of TX is 1, RX is 2.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp-audio.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp-audio.c b/drivers/gpu/drm/imx/hdp/imx-hdp-audio.c
index 48bcc1e..40313e0 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp-audio.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp-audio.c
@@ -196,7 +196,7 @@ void imx_hdp_register_audio_driver(struct device *dev)
 	struct platform_device *pdev;
 
 	pdev = platform_device_register_data(dev, HDMI_CODEC_DRV_NAME,
-					     PLATFORM_DEVID_NONE, &codec_data,
+					     1, &codec_data,
 					     sizeof(codec_data));
 	if (IS_ERR(pdev))
 		return;
-- 
1.7.9.5

